### ISSUE TITLE
Any way to add Structure Reordering?

### ISSUE THREAD

**Issue by bgarrant** [pixelandtonic/CategorySources#5]
I have a bunch of Stucture entries using category fields.  Is there any way to add the Structure reordering function to this plugin?  It shows me the categories and listing perfectly, but just does not let me order them.

**Comment by outline4** [comment]
To my surprise this is still not possible... This would be great to order entries within their respective category... Please give this add-on the love it deserves :)

**Comment by keithmancuso** [comment]
agreed, this would solve so many ux problems we run into with client and seems like it would be a simple add?  but not sure where to start
